### PR TITLE
Admin.php done

### PR DESCRIPTION
**Pull request by Helene-33**

### CODE REVIEW

File: admin.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    
+<?php
+    
+    if(mysqli_num_rows($result) > 0){
+        while($row = mysqli_fetch_assoc($result)){
+            $categoryName = stripslashes($row['CategoryName']);
+            echo '
+                <div class="panel panel-default">
+                <div class="panel-heading">
+                    <h3 class="panel-title">'.$categoryName.' &nbsp;<a href="edit-category.php?CategoryID="><img src="images/edit.ico" alt="edit"></a></h3>
```
Comment (WChihWen) [inline admin.php]: Here we need to post an ID to edit-category.
It is the missing variable needed to post.

```suggestion
                    <h3 class="panel-title">'.$categoryName.' &nbsp;<a href="edit-category.php?CategoryID='.(int)$row['CategoryID'].'"><img src="images/edit.ico" alt="edit"></a></h3>

```

File: admin.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                <ul>';  
+
+                $sqlfeed = 'SELECT * FROM winter2022_rss_feeds WHERE CategoryID='.(int)$row['CategoryID'];
+                $resultfeed = mysqli_query(IDB::conn(),$sqlfeed) or die(trigger_error(mysqli_error(IDB::conn()), E_USER_ERROR));
+
+                if(mysqli_num_rows($resultfeed) > 0){
+                    while($rowfeeds = mysqli_fetch_assoc($resultfeed)){
+                        $myId = $rowfeeds['SubCategory'];
+                        $categoryID = $rowfeeds['CategoryID'];
+                        echo '<li><a href="edit-feeds.php?CategoryName='.$categoryName.'&FeedsID='.(int)$rowfeeds['FeedsID'].'">'.stripslashes($rowfeeds['SubCategory']).' &nbsp; <img src="images/edit.ico" alt="edit"></a></li>';
```
Comment (WChihWen) [inline admin.php]: I missed out on this point before. 
Here we just need to post a FeedsID.

```suggestion
                        echo '<li><a href="edit-feeds.php?FeedsID='.(int)$rowfeeds['FeedsID'].'">'.stripslashes($rowfeeds['SubCategory']).' &nbsp; <img src="images/edit.ico" alt="edit"></a></li>';

```

**Review by WChihWen — CHANGES_REQUESTED** [review]
Submitted a review, please let me know if you have any questions, thanks!

File: admin.php [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+<?php
+
+
+
+//@mysqli_free_result($result);
+
+get_footer(); #defaults to theme footer or footer_inc.php
+
+?>
```
Comment (WChihWen) [inline admin.php]: Once you finish the changes, please click "Re-request review" circle icon next to my name, I will review it, thanks!

image

**Review by Helene-33 — COMMENTED** [review]
I reviewed and committed the changes. I had them on my file but forgot to download them here... Let me know if it worked!

File: admin.php [hunk 4]
```diff
@@ -0,0 +1,82 @@
+<?php
```
Comment (WChihWen) [inline admin.php]: `admin.php` should be in `rss/admin.php`

File: admin.php (hunk 3 above)
Comment (WChihWen) [reply admin.php]: @Helene-33 
When the codes, please click **"Re-request review"** as the screenshot, then I will receive the notification and I will review it, thank you!

**Review by WChihWen — CHANGES_REQUESTED** [review]
The file is in the incorrect folder

**Pull request merged**